aws_lambda_builders/workflows/python_pip/packager.py

```diff
@@ -203,6 +203,9 @@ class DependencyBuilder(object):
         "cp37m": (2, 17),
         "cp38": (2, 26),
         "cp39": (2, 26),
+        "cp310": (2, 26),
```

Comment:
Have we verified installing `llama-cpp-python==0.2.55` works on python 310/311/12 after this change? I recall seeing the same error even after making this change.

Reply:
There was an error if this particular library is not built in container, as the library provides only the source tar so it must be built from source. 

It built fine in container-build without any issue